source/develop/release-management/features/sla/vm-affinity.html.md

```diff
@@ -125,38 +125,80 @@ Filters out host according to affinity enforce mode (hard).
 
 ### REST API
 
-#### Clusters
-
-/GET /api/clusters/{clsuter_id}/affinitygroups <affinity_groups>
-
-` `<affinity_group href="">
-`   `<name></name>
-`   `<description></description>
-`   `<cluster href=""></cluster>
-`   `<positive>`true`</positive>
-`   `<enforcing>`true`</enforcing>
-`   `<members>
-`     `<vm href=""/>
-`     `<vm href=""/>
-`   `</members>
-` `</affinity_group>
-
-</affinity_groups> /POST /api/clusters/{clsuter_id}/affinitygroups <affinity_group></affinity_group> /PUT /api/clusters/{clsuter_id}/affinitygroups/{affinity_group_id} <affinity_group></affinity_group> /DELETE /api/clusters/{clsuter_id}/affinitygroups/{affinity_group_id}
-
-#### Virtual Machines
-
-[not final] GET .../vms
-
-` `<vm>
-`   `<placement_policy>
-         ...
-`     `<affinity_groups>
-`       `<affinity_group href=""/>
-`       `<affinity_group href=""/>
-`     `</affinity_groups>
-         ...
-`   `</placement_policy>
-` `</vm>
-
-[not final]
-
+A new type was added - **AffinityRule** with two attributes of that type,
```

Comment:
Suggested rewrite:
" A new type named AffinityRule, was added [to what?]. It has two attributes: one for virtual machines and another for hosts.